ekncontent/configure.ac

```diff
@@ -120,7 +120,7 @@ AC_SUBST([JASMINE_REPORT_ARGUMENT])
 # Package dependencies
 # Update these whenever you use a function that requires a certain API version
 AX_PKG_CHECK_MODULES([EKN_CONTENT], [glib-2.0 gio-2.0 gobject-2.0 json-glib-1.0 libsoup-2.4],
-    [eos-shard-0 >= 0.1.2 xapian-glib-1.0 >= 1.7])
+    [eos-shard-0 >= 0.1.2 xapian-glib-1.0 >= 1.7 endless-0 >= 0.5])
```

Comment:
did we bump the eos-sdk version? what happened to me is that e-k-l was configured "properly" (with the old eos-sdk I had installed, without profile probes)  but then failed while building ekncontent (because of the missing probe API).

Reply:
Ah, that's a good point; I don't think we did.